res/ari/internal.h

```diff
@@ -59,7 +59,7 @@ struct ast_ari_conf {
 };
 
 /*! Max length for auth_realm field */
-#define ARI_AUTH_REALM_LEN 80
+#define ARI_AUTH_REALM_LEN 256
```

Comment:
This is for ARI and has nothing to do with SIP/PJSIP. If you still want to change it you should update the commit message to reflect it.

Reply:
Sure, and I believe it's crucial to ensure the commit message accurately reflects the intent of this change. I've updated the message accordingly.